### PR TITLE
Creating welcome

### PR DESCRIPTION
**Pull request by jacqueline-abreu**
Branch created for the welcome page.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    display: flex;
+    margin-top: 120px;
+    margin-bottom: 50px;
+}
+
+.area-text{
+    flex: 1;  
+}
+
+.text1{
```
Comment (jhrodriguesit) [inline style.css]:
```suggestion
.welcome-title{
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    font-size: 100px;
+    margin-bottom: 30px;
+}
+
+.text3{
```
Comment (jhrodriguesit) [inline style.css]:
```suggestion
.welcome-description
```

**Review by jhrodriguesit — CHANGES_REQUESTED** [review]

File: style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    height: 480px;
+    margin-left: 190px;
+    margin-right: 40px;
+    margin-top: 150px;
+}
+
+.lobo{
+    flex: 1;
+}
+#text {
```
Comment (jhrodriguesit) [inline style.css]:
```suggestion
#welcome-text {
```

**Comment by jacqueline-abreu** [comment]
I changed the class names and id as request.

**Review by biararaujo — APPROVED** [review]

**Pull request merged**